code/crates/test/framework/src/params.rs

```diff
@@ -16,6 +16,10 @@ pub struct TestParams {
     pub value_payload: ValuePayload,
     pub max_retain_blocks: usize,
     pub stable_block_times: bool,
+    pub enable_discovery: bool,
+    /// Node IDs that should not be added as persistent peers for other nodes
+    /// (simulates nodes that joined after initial network setup)
+    pub exclude_from_persistent_peers: Vec<u64>,
```

Comment:
is this an analog of banning? kinda like dead peers?

Reply:
kind of. It's just for testnet setup so we can add a node to the tesnet without adding its address in the persistent peers of other nodes.